managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedgerFactory.java

```diff
@@ -183,9 +185,24 @@ void delete(String name, CompletableFuture<ManagedLedgerConfig> mlConfigFuture)
      * @throws InterruptedException
      * @throws ManagedLedgerException
      */
+    @Deprecated
     void asyncDelete(String name, CompletableFuture<ManagedLedgerConfig> mlConfigFuture,
```

Comment:
What's our plan to remove this deprecated code?  Can we clean the code to make the caller use the new one?

Reply:
+1